docs/src/ToricVarieties/NormalToricVarieties.md

```diff
@@ -22,118 +22,125 @@ the affine and non-affine case:
 
 ## Constructors
 
+### Affine toric varieties
+
 ```@docs
 AffineNormalToricVariety(C::Cone)
-CyclicQuotientSingularity
 NormalToricVariety(C::Cone)
-NormalToricVariety(PF::PolyhedralFan)
-NormalToricVariety(P::Polyhedron)
-toric_projective_space
-hirzebruch_surface
-del_pezzo
+AffineNormalToricVariety(v::NormalToricVariety)
 ```
 
+### Cyclic Quotient Singularities
 
-## Properties of toric varieties
+Cyclic quotient singularities are quotients of $\mathbb{C}^2$ by the action of
+$\mathbb{Z}/n\mathbb{Z}$ acting via 
+$$\left(\begin{array}{cc}\xi & 0\\0 & \xi^q\end{array}\right)$$,
+where $\xi$ is a $n$-th root of unity, and $0<q<n$ are two coprime integers.
 
-```@docs
-hastorusfactor
-isaffine
-iscomplete
-isfano
-isgorenstein
-issimplicial
-issmooth
-isnormal
-isorbifold
-isprojective
-isq_gorenstein
-```
+For the notation we rely on [Chr91](@cite) and [Ste91](@cite).
 
+!!! warning
+    Note that the notion of Hirzebruch-Jung continued fraction differs from the
+    commonly known continued fraction.
```

Comment:
This remarks is a bit "out of the blue" here. I assume it makes sense if one has read the two citations you give above? Then I'd mention this, e.g.
> CITE and CITE use Hirzebruch-Jung continued fractions, which differ from the usual  continued fractions found in most literature, and also used by the rest of OSCAR.

Reply:
Ok for me, but again written by @lkastner. @lkastner are you ok with this change?

Reply:
Sounds good to me, so I make this concrete:
```suggestion
    Note that [Chr91](@cite) and [Ste91](@cite) use Hirzebruch-Jung continued fraction, which differ from the
    commonly known continued fraction from literature and used in the rest of OSCAR.
```

Reply:
Thank you Lars.